src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Text_Format_PowerFxV1CompatDisabled.txt

```diff
@@ -1,25 +1,25 @@
 #SETUP: disable:PowerFxV1CompatibilityRules
 
->> Text(123.466, "[$-en-US]$#0.0M") 
+>> Text(123.466, "[$-en-US]$#0.0M")
 Error({Kind:ErrorKind.InvalidArgument})
 
 >> Text(1, "M#")
 Error({Kind:ErrorKind.InvalidArgument})
 
 // If have empty language code, throw error
 >> Text(1234.5678,"[$-]")
-Error({Kind:ErrorKind.InvalidArgument})
+Error({Kind:ErrorKind.BadLanguageCode})
 
 // If not closed by a ']', throw error
 >> Text(1234.5678,"[$-a")
-Error({Kind:ErrorKind.InvalidArgument})
+Error({Kind:ErrorKind.BadLanguageCode})
 
 // If not closed by a ']', throw error
 >> Text(1234.5678, "[$-en-US#,##0.00", "en-US")
-Error({Kind:ErrorKind.InvalidArgument})
+Error({Kind:ErrorKind.BadLanguageCode})
 
 >> Text(1234.5678, "[$-]fr-FR # ##0,00", "vi-VI")
-Error({Kind:ErrorKind.InvalidArgument})
+Error({Kind:ErrorKind.BadLanguageCode})
 
 >> Text(1234.5678, "[$-fr-FR][$-en-US] # ##0,00", "vi-VI")
```

Comment:
>[$-fr-FR][$-en-US] [](http://example.com/codeflow?start=20&length=18)

Can we add a similar test case (with more than 1 language code) to the `Text_Format.txt` file?

Reply:
no need to duplicate it in text_format. This one need to be here because when runtime and compile return different type of error, we need it in either text_format_powerfxv1compat or text_format_powerfxv1compatdisabled to check different error for the same case.